orc8r/tools/ansible/roles/pkgrepo/tasks/debian.yml

```diff
@@ -23,9 +23,15 @@
   #  when: preburn
 
 - name: Add JFrog key
-  apt_key:
-    data: "{{ lookup('file', 'jfrog.pub') }}"
-    state: present
+  block:
```

Comment:
As discuss, I think we can do it one shot with apt_key and specifying the url. [ansible module for apt](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/apt_key_module.html) 

```bash
- name: Add an Apt signing key, uses whichever key is at the URL
  ansible.builtin.apt_key:
    url: https://ftp-master.debian.org/keys/archive-key-6.0.asc
    state: present
```